entgql/template.go

```diff
@@ -304,6 +304,9 @@ func mutationInputs(nodes []*gen.Type) ([]*MutationDescriptor, error) {
 func filterNodes(nodes []*gen.Type, skip SkipMode) ([]*gen.Type, error) {
 	filteredNodes := make([]*gen.Type, 0, len(nodes))
 	for _, n := range nodes {
+		if n.IsEdgeSchema() {
```

Comment:
We want to skip edge schemas with composite primary keys, right? Not all edge schemas.

Reply:
Cool.